README.md

```diff
@@ -1,290 +0,0 @@
-wolox-react-bootstrap
```

Comment:
Everything above the `## Components` section keep it here. The other stuff leave it in the new project. In the future we'll see how to improve this